### ISSUE TITLE
pagination

### ISSUE THREAD

**Issue by rzhbishal** [MahirMahdi/refine-jsonapi#3]
is it possible to create json api using this format?

https://jsonapi.org/examples/

**Comment by MahirMahdi** [comment]
Yes, i've followed this documentation to create refine's json api
